src/lsdb/core/search/pixel_search.py

```diff
@@ -18,9 +19,30 @@ class PixelSearch(AbstractSearch):
     Does not filter points inside those partitions.
     """
 
-    def __init__(self, pixels: list[tuple[int, int]]):
+    def __init__(self, pixels: tuple[int, int] | HealpixPixel | list[tuple[int, int] | HealpixPixel]):
         super().__init__(fine=False)
-        self.pixels = [HealpixPixel(o, p) for o, p in set(pixels)]
+        if isinstance(pixels, tuple):
+            self.pixels = [HealpixPixel(pixels[0], pixels[1])]
+        elif isinstance(pixels, HealpixPixel):
+            self.pixels = [pixels]
+        elif pd.api.types.is_list_like(pixels):
+            if len(pixels) == 0:
+                raise ValueError("Some pixels required for PixelSearch")
+            self.pixels = list({get_healpix_pixel(pix) for pix in pixels})
```

Comment:
Why cast through `set` on the way to `list`? Duplicates are expected?

Reply:
It was restricting to set previously, so I maintained the behavior, but it doesn't really matter for the `filter_from_pixel_list` call. Removed.